### PR TITLE
ARXIVCE-742: 3 locations where we could put a message

### PR DESCRIPTION
**Pull request by bmaltzan**
for the user about being in gcp.

### CODE REVIEW

**Review by bdc34 — APPROVED** [review]

File: browse/templates/base.html
Comment (SBBCornell) [inline browse/templates/base.html:39]: @alisonhofer might have a different take, but arXiv's tone is usually more matter of fact. What about "Moving to the cloud" or "Server stability update"?

File: browse/templates/home/special-message.html
Comment (SBBCornell) [inline browse/templates/home/special-message.html:28]:
```suggestion
  <p>arXiv is preparing for our next 30 years by transitioning from our current aging infrastructure to stable cloud-hosted services. </p>
```

File: browse/templates/home/special-message.html
Comment (SBBCornell) [inline browse/templates/home/special-message.html:27]:
```suggestion
  <span class="label">Cloud transition</span>
```

File: browse/templates/user_banner.html
Comment (SBBCornell) [inline browse/templates/user_banner.html:32]:
```suggestion
    <p><a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/1l">Report an issue</a></p>
```

File: browse/templates/user_banner.html
Comment (SBBCornell) [inline browse/templates/user_banner.html:31]:
```suggestion
    <p>We're making arXiv even more stable by transitioning to the cloud.</p>
```

**Review by SBBCornell — COMMENTED** [review]
Not sure if you were looking for word-smithing, but I made some suggestions. 

One other comment is that having both the main banner and the minimal banner seems like a lot of messaging. The minimal banner is great if we can get the message across in <5 words. But if not, the main banner gives us room for a whole sentence or two!

File: browse/templates/home/special-message.html
Comment (bdc34) [reply browse/templates/home/special-message.html:28]: I'd suggest "arxiv is preparing for our next 30 years by transitioning to cloud-hosted services"

Comment (cbf66) [reply browse/templates/home/special-message.html:28]: Let's not put this out there until we have a landing page that explains the plan more fully. I don't want to introduce the whole idea of the cloud transition with a one sentence message.

**Review by cbf66 — COMMENTED** [review]
Please hold this until I have a landing page ready. Give me an hour.

**Comment by bmaltzan** [comment]
> Not sure if you were looking for word-smithing, but I made some suggestions.

Definitely.
Mine was just some random placeholder text.

**Comment by bdc34** [comment]
I changed the base to `develop` the gcp-xyz branches are not in use any more. I'll delete them soon.

**Comment by cbf66** [comment]
> Please hold this until I have a landing page ready. Give me an hour.

Done [https://info.arxiv.org/help/cloud.html](url)

**Pull request merged**
